client/src/hooks/useCafe.ts

```diff
@@ -0,0 +1,11 @@
+import client from '../client';
+import useSuspenseQuery from './useSuspenseQuery';
+
+const useCafe = (cafeId: number) => {
```

Comment:
```suggestion
const useCafe = (cafeId: Cafe['id']) => {
```
`Cafe`에서 파생된 id 타입을 쓰는 것이 적절해보입니다!!